tests/profiler/user_workflow_fixtures.py

```diff
@@ -0,0 +1,194 @@
+from typing import List
+
+import pytest
+
+from great_expectations.core import ExpectationConfiguration, ExpectationSuite
+
+# TODO: Move these fixtures to integration tests
+
+@pytest.fixture(scope="module")
+def alice_columnar_table_single_batch():
+    profiler_config = """
+variables:
+  max_user_id: 999999999999
+  min_timestamp: 2004-10-19 10:23:54
+rules:
+  my_rule_for_user_ids:
+    domain_builder:
+      class_name: SimpleSemanticTypeColumnDomainBuilder
+      semantic_types:
+        - user_id
+    parameter_builders:
+      - name: my_min_user_id
+        class_name: MetricParameterBuilder
+        metric_name: column.min
+    expectation_configuration_builders:
```

Comment:
Does the configuration for an `expectation_configuration_builder` need a `class_name` (for instantiation)?..

Reply:
See my explanation above about `Rule`

Reply:
This is something that I think we still might need to settle. I posted alternate examples under the Heidi and Ivan user stories. We'll need to converge on one of the different configurations that are possible here.

Reply:
@anthonyburdi @talagluck Please see my comment above.  I think that relying on `instantiate_class_from_config()` consistently is worth consideration, so let us discuss at a next opportunity.  Thank you.

Reply:
@alexsherstinsky - totally agree. In the `verbose_profiler_config` I added `class_name` and `module_name` for all of the builder classes. In the `simplified_profiler_config` is a proposal for a simplified version which assumes some defaults @talagluck I agree we need to discuss Heidi and Ivan - but I think even if we do go with either we should still support the verbose option of specifying a `class_name` and `module_name` for each `expectation_configuration_builder`

Reply:
@anthonyburdi @talagluck I think that brevity is fine as long as we do not deviate from relying on `instantiate_class_from_config()` -- so that we get an object for any class whose configuration appears in the YML file (it is when we deviate from this process and instantiate YAML-configurable classes explicitly via Python is when we get into difficulties).